client/extensions/woocommerce/app/settings/email/email-settings/components/notifications-origin.js

```diff
@@ -41,32 +39,30 @@ const NotificationsOrigin = ( {
 	const placeholderComponent = <p className="components__is-placeholder" />;
 
 	return (
-		<ListItem>
-			<ListItemField className="components__notification-origin">
-				{ ! isPlaceholder ? <FormLabel>{ item.title }</FormLabel> : placeholderComponent }
-				<FormTextInput
-					className={ isPlaceholder ? 'components__is-placeholder' : null }
-					isError={ emailValidationError }
-					name={ item.field }
-					onChange={ change }
-					value={ recipient }
-					placeholder={ placeholder }
+		<div className="components__notification-origin">
+			{ ! isPlaceholder ? <FormLabel>{ item.title }</FormLabel> : placeholderComponent }
+			<FormTextInput
+				className={ isPlaceholder ? 'components__is-placeholder' : null }
+				isError={ emailValidationError }
+				name={ item.field }
+				onChange={ change }
+				value={ recipient }
+				placeholder={ placeholder }
+			/>
+			{ emailValidationError && (
+				<FormTextValidation
+					isError={ true }
+					text={ translate( '%(recipient)s is not a valid email address.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( '%(email)s is not a valid email address.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).